.github/workflows/ldap-integration.yaml

```diff
@@ -0,0 +1,45 @@
+name: External IDP Integration Tests
+
+on:
+  pull_request:
+    branches:
+    - master
+
+# This ensures that previous jobs for the PR are cancelled when the PR is
+# updated.
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  ldap-test:
+    name: LDAP Tests with Go ${{ matrix.go-version }}
+    runs-on: ubuntu-latest
+
+    services:
+      openldap:
+        image: donatello/openldap:1
```

Comment:
we can push this to quay.io/minio/openldap - do you want to do that?  - just pushed `quay.io/minio/openldap`